src/Akeneo/Pim/Automation/DataQualityInsights/front/src/application/component/Dashboard/KeyIndicators/ProductMessageBuilder.tsx

```diff
@@ -49,13 +53,9 @@ export const ProductMessageBuilder = (props: Props) => {
   ].filter(Boolean);
 
   const markersMapping: MarkersMapping = {
-    '<button_a/>': buttonA,
-    '<button_b/>': buttonB,
+    '<improvable_products_count_link/>': buttonA,
+    '<improvable_product_models_count_link/>': buttonB,
```

Comment:
We loose the context when we are using names like buttonA and buttonB 
What do you think about the following suggestion?
```suggestion
    '<improvable_products_count_link/>': nbProductsKO > 0 ? productsButton : undefined,,
        '<improvable_product_models_count_link/>': nbProductModelsKO > 0 ? productModelsButton : undefined,
```

Reply:
Could not apply the suggestion, I added a comment